lib/elixir/lib/enum.ex

```diff
@@ -1542,18 +1542,25 @@ defmodule Enum do
   ## Examples
 
       # Although not necessary, let's seed the random algorithm
-      iex> :rand.seed(:exsplus, {1, 2, 3})
-      iex> Enum.random([1, 2, 3])
-      2
+      iex> :rand.seed(:exsplus, {9, 3, 4})
```

Comment:
Why this change? Users will never seed this by hand (hopefully), so I think it may even be better to use `{1, 2, 3}` to make that even clearer?